logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/ConversationEventReceiver.kt

```diff
@@ -57,8 +60,13 @@ class ConversationEventReceiver(
                         Message.Status.SENT
                     )
                     //TODO Multiplatform logging
-                    println("Message received: $message")
-                    messageRepository.persistMessage(message)
+                    kaliumLogger.i(message = "Message received: $message")
+                    when (message.content) {
+                        is MessageContent.Text -> messageRepository.persistMessage(message)
+                        is MessageContent.DeleteMessage -> messageRepository.softDeleteMessage(messageUuid = message.content.messageId, message.conversationId)
```

Comment:
So the problem is this: 
What prevents User B from sending a `DeleteMessage` which deletes a message sent by User A. AFAIK nothing prevents this from happening right now so we need to have check somewhere which validate that `DeleteMessage` was sent by the original sender of the message which should be deleted.